pyomo/contrib/mindtpy/tests/MINLP2_simple.py

```diff
@@ -54,7 +54,7 @@ class SimpleMINLP(ConcreteModel):
 
     def __init__(self, *args, **kwargs):
         """Create the problem."""
-        kwargs.setdefault('name', 'DuranEx1')
+        kwargs.setdefault('name', 'SimpleMINLP2')
```

Comment:
It might help future you to come up with a more descriptive name for this problem.

Reply:
I agree, though these were arbitrary examples given in the Duran and Grossmann's paper.